custom_components/powercalc/data/ikea/LED2106R3/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TAPO P110",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 4,
+        "SLEEP_TIME": 5,
+        "VERSION": "v1.10.0:docker"
+    },
+    "name": "TRADFRI bulb GU10 WS 345 lm, dimmable, white spectrum",
+    "standby_power": 0.1
```

Comment:
Were you able to measure this?
0.1 is very small. Non of the smart bulbs in powercalc library is using that little. Zigbee ikea bulbs should typically be 0.2-0.3W

Reply:
What did you use as dummy load. Only full resistant loads are reliable for that. You'll get unreliable readings when using a LED light as dummy load for example.

Reply:
LED Light with stable 8,69W to 8,60W measured more time with same result

Reply:
I'm not sure why you also needed a dummy load when also using 3 bulb. The latter should be sufficient to measure.
I am not an expert on the topic, but there were valid discussions in the past as to why LED lights could not be used as an dummy load.
You measurements looks ok to me, except the standby power which is never below 0.2W for zigbee lights.
Maybe @nepozs can join in on this topic.

Reply:
The Ikea website is also mentioning 0.2W standby power. I am ok setting both light to 0.2W which is a better estimate than 0.1W
https://www.ikea.com/nl/nl/p/tradfri-led-lamp-gu10-345-lumen-smart-draadloos-dimbaar-wit-spectrum-50547413/

Reply:
No problem, btw I tried only with 3 pcs of this bulbs but it is not enought for measurement.